src/dependencies.scala

```diff
@@ -121,6 +121,29 @@ object Dependencies {
     }
   }
 
+  case class LibEquals[A <: AndroidLibrary](lib: A)
+  {
+    override def equals(other: Any) = {
+      (lib, other) match {
+        case (l @ AarLibrary(_), LibEquals(r @ AarLibrary(_))) ⇒
+          l.moduleID == r.moduleID
+        case (l @ LibraryProject(_), LibEquals(r @ LibraryProject(_))) ⇒
+          l.path.getCanonicalFile == r.path.getCanonicalFile
```

Comment:
mentioned on irc: should identify AutoLibraryProject vs. non-AutoLibraryProject and fail if canonical paths ==